src/Assets/LightSkyDefense/Scripts/Credit.cs

```diff
@@ -17,7 +23,11 @@ private void OnHandHoverBegin(Hand hand)
     {
         var player = Player.instance.gameObject;
         var creditOwner = player.GetComponent<CreditOwner>();
-        creditOwner?.AddCredits(value);
+        if (creditOwner == null) return;
```

Comment:
Can we add a new line around if statements please.

```suggestion

        if (creditOwner == null) return;

```